src/data/cameraData.ts

```diff
@@ -0,0 +1,30 @@
+interface cameraDetails {
```

Comment:
This should be named in PascalCase
```suggestion
interface CameraDetails {
```
remember to rename it everywhere below!